### PR TITLE
Update website configuration for GitHub Pages compatibility

### PR DESCRIPTION
**Pull request by KeithMoyer**
This pull request does multiple things:
 - Updates instructions to test using container image designed for testing GitHub Pages.
 - Removes the Gemfiles, as they're not used in GitHub Pages and cause issues when testing against environments that simulate it.
 - Cleans up some places where we were creating temporary files in the repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,44 @@
...
 
 ```bash
-# First time: (slow)
-docker run --rm --label=asylo-jekyll --volume=$(pwd):/srv/jekyll -v $(pwd)/_site:$(pwd)/_site -it -p 127.0.0.1:4000:80 jekyll /jekyll:3.6.0 sh -c "bundle install && rake test && bundle exec jekyll serve --incremental --host 0.0.0.0"
-# Then open browser with url 127.0.0.1:4000 to see the change.
-# Subsequent, each time you want to see a new change and you stopped the previous run by ctrl+c: (much faster)
-docker start asylo-jekyll -a -i
-# Clean up, only needed if you won't be previewing website changes for a long time or you want to start over:
-docker rm asylo-jekyll
+docker run -it --rm -p 4000:4000 -v ${PWD}:/srv/jekyll jekyll:jekyll:pages \
```
Comment (deeglaze) [inline README.md]: jekyll/jekyll:pages

File: README.md [hunk 2]
```diff
@@ -1,46 +1,44 @@
...
+# Then open browser with url localhost:4000 to see the change.
 ```
 
-The `rake test` part is to make sure you are not introducing HTML errors or bad links, you should see
+Some tests are included to make sure you are not introducing HTML errors or bad
+links.
 
 ```bash
-HTML-Proofer finished successfully.
+docker run -it --rm -v ${PWD}:/srv/jekyll jekyll:jekyll:pages rake test
```
Comment (deeglaze) [inline README.md]: jekyll/jekyll:pages

Comment (deeglaze) [reply README.md]: This command simply hangs for me. I just see this
rm -fr _rakesite
mkdir _rakesite
bundle exec jekyll build --config _config.yml,_rake_config_override.yml

File: README.md [hunk 3]
```diff
@@ -1,46 +1,44 @@
...
 ```
 
-in the output
-
-> In some cases the `--incremental` may not work properly and you might have to remove it.
-
-Alternatively, if you just want to develop locally w/o Docker/Kubernetes/Minikube, you can try installing Jekyll locally.
-You may need to install other prerequisites manually (which is where using the docker image shines). Here's an example of doing
-so for Mac OS X:
+Alternatively, if you just want to develop locally w/o
```
Comment (deeglaze) [inline README.md]: s#w/o#without#

File: README.md [hunk 4]
```diff
@@ -58,22 +56,25 @@ bundle exec jekyll serve
 
 ## Linting
 
-You should run `scripts/linters.sh` prior to checking in your changes.
-This will run 3 tests:
+You should run `scripts/linters.sh` prior to checking in your changes. This will
```
Comment (deeglaze) [inline README.md:59]: This is the first I've seen linters.sh. That script should be documented in scripts' README.md with its prerequisites. How does one acquire mdspell, for example?

**Review by deeglaze — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (KeithMoyer) [reply README.md]: Are you running it using the content from the pull request?  That output implies that you are not.

File: README.md (hunk 4 above)
Comment (KeithMoyer) [reply README.md:59]: No idea.  I didn't write this text; I only ran a markdown formatter on it. If we want to clean up or remove these part, let's do it separate from this unrelated pull request.

File: README.md (hunk 1 above)
Comment (KeithMoyer) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (KeithMoyer) [reply README.md]: Done

**Review by deeglaze — APPROVED** [review]

**Pull request merged**